Allow overwriting external functions (with ``calldata`` arguments) with public functions (with ``memory`` arguments) (#5445)
ethereum/solidity: Changelog.md

### 0.5.1 (unreleased)

Language Features:

>  * Inheritance: Allow overwriting external functions with ``calldata`` arguments with public functions with ``memory`` arguments.

Reviewer: ```suggestion
 * Inheritance: Allow public functions to overwrite external functions that have ``calldata`` parameters.
```
